charts/buffer/templates/ingress/ingress.yaml

```diff
@@ -1,23 +1,8 @@
-{{ if .Values.ingress.enabled -}}
-apiVersion: networking.k8s.io/v1beta1
-kind: Ingress
-metadata:
-  name: {{ .Values.name }}-ingress
-  labels:
-    {{- include "buffer.labels" . | nindent 4 }}
-  annotations:
-    networking.gke.io/v1beta1.FrontendConfig: "{{ .Values.name }}-ingress-frontendconfig"
-    kubernetes.io/ingress.global-static-ip-name: {{ required "ingress.staticIpName value is required" .Values.ingress.staticIpName | quote }}
-    kubernetes.io/ingress.allow-http: "false"
-{{- if not (empty .Values.ingress.cert.preSharedCerts) }}
-    ingress.gcp.kubernetes.io/pre-shared-cert: {{ .Values.ingress.cert.preSharedCerts | join "," | quote }}
-{{- end }}
+{{- include "libchart.ingress" (list . "buffer.ingress" )  -}}
+{{- define "buffer.ingress" -}}
 spec:
 {{- if empty .Values.ingress.cert.preSharedCerts }}
   tls:
```

Comment:
should this logic be added to the `ingresslib` chart? I know it's repeated in a lot of places :)